### PR TITLE
Add sentry configuration

### PR DESCRIPTION
**Pull request by smeny**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,6 +49,9 @@ gem "aws-sdk-s3"
 gem "paper_trail"
 # Store active record objects in version whodunnits
 gem "paper_trail-globalid"
+# Receive exceptions and configure alerts
+gem "sentry-rails", "~> 5.0", ">= 5.0.2"
```
Comment (baarkerlounger) [inline Gemfile]: Do we need the min and the twiddle waka?

Comment (smeny) [reply Gemfile]: Gone

File: config/initializers/sentry.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Sentry.init do |config|
+  config.breadcrumbs_logger = %i[active_support_logger http_logger]
+
+  # To activate performance monitoring, set one of these options.
+  # We recommend adjusting the value in production:
+  config.traces_sample_rate = 0.5
```
Comment (baarkerlounger) [inline config/initializers/sentry.rb:4]: Do we want to be configuring environments in here? Something like:

```
config.environments = %w[staging production]
```

Comment (smeny) [reply config/initializers/sentry.rb:4]: It is not needed, a lot is actually fetched from Rails.
I would leave the default which uses RAILS_ENV.
Happy to add the other one.

**Review by baarkerlounger — APPROVED** [review]

File: config/initializers/sentry.rb (hunk 2 above)
Comment (baarkerlounger) [reply config/initializers/sentry.rb:4]: Sorry that was just a question. Happy to leave the default as well.

**Pull request merged**